search-service/src/test/kotlin/com/egm/stellio/search/service/EntityPayloadServiceTests.kt

```diff
@@ -586,16 +586,14 @@ class EntityPayloadServiceTests : WithTimescaleContainer, WithKafkaContainer {
                     now
                 )
             }
-
         entityPayloadService.updateTypes(entity01Uri, listOf(APIARY_TYPE), ngsiLdDateTime(), false)
+            .shouldSucceed()
+
+        entityPayloadService.retrieve(entity01Uri)
             .shouldSucceedWith {
-                assertFalse(it.isSuccessful())
-                assertEquals(1, it.notUpdated.size)
-                val notUpdatedDetails = it.notUpdated[0]
-                assertEquals(JSONLD_TYPE, notUpdatedDetails.attributeName)
                 assertEquals(
-                    "A type cannot be removed from an entity: [$BEEHIVE_TYPE] have been removed",
-                    notUpdatedDetails.reason
+                    listOf(BEEHIVE_TYPE, APIARY_TYPE),
```

Comment:
the SQL request is actually working:

<img width="1136" alt="image" src="https://github.com/stellio-hub/stellio-context-broker/assets/175382/39fec6fa-ba4b-4069-b0c1-b57fa3c6a990">

the `types` column is correctly updated, it is the `type` in the payload that is not updated (see lines 427-431 in `EntityPayloadService.kt`)